src/components/Footer/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Box, IconButton } from '@material-ui/core'
+import GitHubIcon from '@mui/icons-material/GitHub'
```

Comment:
Icons can be imported from MUI Icons v4 (https://v4.mui.com/components/material-icons/):
import GitHubIcon from '@material-ui/icons/GitHub';
etc.